server/src/main/java/org/apache/iotdb/db/mpp/plan/execution/config/FlushTask.java

```diff
@@ -54,10 +54,10 @@ public ListenableFuture<ConfigTaskResult> execute(IConfigTaskExecutor configTask
       tFlushReq.setIsSeq(flushStatement.isSeq().toString());
     }
     IoTDBConfig config = IoTDBDescriptor.getInstance().getConfig();
-    if (flushStatement.isLocal()) {
-      tFlushReq.setDataNodeId(config.getDataNodeId());
-    } else {
+    if (flushStatement.isCluster()) {
       tFlushReq.setDataNodeId(-1);
+    } else {
+      tFlushReq.setDataNodeId(config.getDataNodeId());
```

Comment:
If SQL is `FLUSH` in standalone mode, then `flushStatement.isCluster() == true` in your ASTVisitor modificaton. So what will happen here?

Reply:
Standalone mode won’t check it, only flush the data in StorageEngine.

Reply:
Got it.